source/MaterialXTest/MaterialXGenShader/GenShader.cpp

```diff
@@ -165,6 +165,47 @@ TEST_CASE("GenShader: Graph + Nodedf Translation Check", "[genshader]")
     mx::writeToXmlFile(doc, "transparency_test_nodedefs.mtlx");
 }
 
+TEST_CASE("GenShader: Shader Translation", "[translate]")
```

Comment:
Isn't there already a translation test so this would be redundant ?

Reply:
I couldn't find anything that tested translation API so added this. Can you point me to an existing test, perhaps I can update it to add usdpreviewsurface translation.